torch/export/unflatten.py

```diff
@@ -779,16 +782,7 @@ def __init__(
 
         self.parent_call_module: Optional[torch.fx.Node] = None
         if parent is not None:
-            if self.fqn in module_call_graph and num_calls == 1:
-                raise ValueError(
-                    f"Cannot unflatten multiple calls to module {self.fqn} while preserving its signature "
-                    "because each of these calls might have generated a different specialized graph. "
-                    f"If the reason you want to preserve the signature is to swap {self.fqn} with another module, "
-                    "consider using _swap_modules() directly on the exported program instead of unflattening it."
-                )
-            # generate call name for self.fqn
-            child_fqn = _call_name(self.fqn, num_calls + 1)
-            accessor = _compute_accessor(parent.fqn, child_fqn)
+            accessor = _compute_accessor(parent.fqn, self.child_fqn)
```

Comment:
Does the accesor now can have @ in their name?

Reply:
Accessor already potentially had `@` in the name, just moved `child_fqn` to an attribute so that it can be accessed later in `self.finalize_outputs(...)` without recomputing.